### PR TITLE
Add info on how to debug lua filters

### PR DESCRIPTION
**Pull request by iandol**
The mobdebug package and Zerobrane studio allow Lua filters to be debugged with a REPL and full control. This is really useful for those of us writing filters who are not so proficient in Lua or don't fully understand the AST structures exposed via the filter.

### CODE REVIEW

**Review by alerque — APPROVED** [review]

File: doc/lua-filters.md [hunk 1]
```diff
@@ -305,6 +305,24 @@ end
...
+can be run from the terminal, it is more useful run within the 
+donation-ware Lua editor and IDE, [Zerobrane](https://studio.zerobrane.com/). 
+Zerobrane offers a REPL console and UI to step-through and view all 
+variables and state.
+
+If you already have Lua 5.3 installed, you can add `mobdebug` and its 
+dependency `luasocket` using [`luarocks`](https://luarocks.org), which 
+should then be available on the path. Zerobrane also includes both of these 
+in its package, so if you don't want to install Lua seperately, you should 
+modify your lua path and cpath to include the correct locations;
```
Comment (tarleb) [inline doc/lua-filters.md:323]: Small typo: Lua should be capitalized; or maybe mention the environment variables directly?

**Review by tarleb — APPROVED** [review]
:heart:

File: doc/lua-filters.md (hunk 1 above)
Comment (iandol) [reply doc/lua-filters.md]: Hi @tarleb, I've added the correct env names, and also linked to the dependent packages on luarocks. I am hard-wrapping the paragraphs at 80 chars, is that OK?

Comment (tarleb) [reply doc/lua-filters.md]: Great, thanks!

80 chars is ok. Most of the text uses a limit of 66, but anything below 80 works fine.

**Pull request merged**